docker/user.sh

```diff
@@ -1,24 +1,47 @@
 #!/bin/bash
 
-# Copyright (c) 2022 - 2022, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
-groupmod -o -g $USER_GID macaron
-usermod -o -g $USER_GID -u $USER_UID macaron
+# We update the GID and UID of the existing macaron user in the container
+# with the GID and UID of the current user in the host machine.
+# This will make the files created during the runtime of the container would have
+# the current user of the host machine as the owner.
+if [[ -n "$USER_GID" ]] && [[ -n "$USER_UID" ]];
+then
+    groupmod -o -g "$USER_GID" macaron
```

Comment:
Can we use the long cmd options to improve the readability?

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/37e91e824c2d7a67e0071df08ce83048e0c4ab52